### ISSUE TITLE
Setting customColors questions

### ISSUE THREAD

**Issue by CoralSilver** [SC5/sc5-styleguide#921]
I'm not familiar with PostCSS.  Must the file that defines custom UI color overrides be a css file instead of Sass file now?

Do we redefine variables written like this: ```$primary-color: #1C3849 !default;``` or this: ```--primary-color: #1C3849 !default;
``` or both?

**Comment by junaidrsd** [comment]
Yes, it should be a css file. You don't need to redefine variables, it accepts sass-like syntax.
